src/clientManager.js

```diff
@@ -3,13 +3,31 @@ var fs = require("fs");
 var Client = require("./client");
 var mkdirp = require("mkdirp");
 var Helper = require("./helper");
+var path = require("path");
+
+var USERS_PATH = path.join(Helper.HOME, "users");
```

Comment:
"static" methods are hidden from the prototype chain, so `this.USERS_PATH` would be `undefined`. I could do `var USERS_PATH = ClientManager.USERS_PATH = path....` though